Packages/MIES/MIES_PulseAveraging.ipf

```diff
@@ -743,6 +743,11 @@ static Function [STRUCT PulseAverageSetIndices pasi] PA_GenerateAllPulseWaves(st
 				sweepList = AddListItem(sweepNoStr, sweepList, PA_PROPERTIES_STRLIST_SEP, inf)
 			endif
 
+			WAVE/Z activeHS = OVS_ParseIgnoreList(win, sweepNo=sweepNo)
+			if(WaveExists(activeHS) && activeHS[region] == 0)
+				continue
+			endif
+
```

Comment:
I've not played around with it. This looks like to be technically the correct solution. Nice that it is so short. But isn't that dead slow as you call that for every pulse?

Reply:
I'm taking over this PR.
- Parse ignore list on input to OVS and store parsed in waves
- Waves need to be cheap to query with sweepNo and experiment from PA_GenerateAllPulseWaves